src/python/pants/base/run_info.py

```diff
@@ -92,12 +92,3 @@ def add_basic_info(self, run_id, timestamp):
             ("buildroot", buildroot),
             ("version", VERSION),
         )
-
-    def add_scm_info(self):
```

Comment:
Unused. FYI @asherf, this is the Pants's run info, but isn't being wired up.

Reply:
we use it in our plugin....

Reply:
I am fine with removing it, but I suggest holding off on this particular change so we can port this logic into our plugin after the scm change is released.

Reply:
I'm fine keeping it. I'll add it back.